tests/unit/transforms/attachment-test.js

```diff
@@ -0,0 +1,62 @@
+import { moduleFor, test } from 'ember-qunit';
+
+import Ember from 'ember';
+
+let testSerializedData = {
+  'test.txt': {
+    content_type: 'text/plain',
+    data: 'hello world!'
+  },
+  'stub.json': {
+    stub: true,
+    content_type: 'application/json'
+  }
+};
+
+let testDeserializedData = [
+  Ember.Object.create({
+    name: 'test.txt',
+    content_type: 'text/plain',
+    data: 'hello world!'
```

Comment:
technically this should be base64-encoded (e.g. `btoa('hello world')`). although I don't think it matters for the purpose of the test